clang/include/clang/Basic/DiagnosticCommonKinds.td

```diff
@@ -73,7 +73,7 @@ def warn_pragma_debug_unexpected_argument : Warning<
 def warn_fp_nan_inf_when_disabled : Warning<
   "use of %select{infinity|NaN}0%select{| via a macro}1 is undefined behavior "
   "due to the currently enabled floating-point options">,
-  InGroup<DiagGroup<"nan-infinity-disabled">>;
+  InGroup<DiagGroup<"NanInfDisabledDocs">>;
```

Comment:
```suggestion
  InGroup<DiagGroup<"nan-infinity-disabled", NanInfDisabledDocs>>;
```
You were naming the warning group `NanInfDisabledDocs`, you still need the group name. This suggests we're missing test coverage because no tests broke despite the warning group name changing.

Reply:
Thanks @AaronBallman. I have added a few other lines in the LIT test. I can't think of anything else.